common/logging/logger.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from common.config_reader import ConfigReader
+import copy
+import json
+import os
+
+
+class Logger:
+    __config_filename = "log_config.json"
+    __log_filename = ""
+    __config_type = ""
+    data = ""
+
+    def __init__(self):
+
+        file_name = self.__config_filename
+
+        maps_path = os.path.join(ConfigReader.get_data('base_path'), 'data/configs/app_config/{}'.format(file_name))
+        with open(maps_path, 'r') as f:
+            __data = json.load(f)
+
+        data = copy.deepcopy(__data)
+
+        __log_filename = data["log_filename"]
+        __config_type = data["config_type"]
+
+    @staticmethod
+    def get_log_file_name():
+        return Logger.__log_filename
+
```

Comment:
make the log file name for every session different. Can be a timestamp with any prefix defined in the cofigs